panic in svggen

Hello

When trying to convert the school-bus.svg into vector, the tool crashes and doesn't give any impression on what's wrong - I don't think it's the input file as I've tried a lot simpler svg's as well:

```
$ ~/go/bin/svggen -i school-bus.svg -o bus-vector.go
panic: 1:12: expected ';', found '-'

goroutine 1 [running]:
main.main()
        /home/fas/go/pkg/mod/github.com/inkeliz/giosvg@v0.0.0-20220907230924-ba74fc8e323f/cmd/svggen/main.go:206 +0xa0d

```

--- a/cmd/svggen/main.go
+++ b/cmd/svggen/main.go
@@ -48,9 +48,12 @@ func main() {
 		if output != "" {
 			abs, _ := filepath.Abs(output)
 			pkg = filepath.Base(filepath.Dir(abs))
+			pkg = strings.Replace(pkg, "", "_", -1)
+			pkg = strings.Replace(pkg, "-", "_", -1)
 		} else {
 			pkg = "assets"
 		}
+		fmt.Printf("package name not defined, use -pkg flag to set a custom name. Using %s as name instead. \n", pkg)
 	}
 
 	out := bytes.NewBuffer(nil)

